make_rpm

```diff
@@ -124,15 +124,15 @@ if [[ $skip_build == 0 ]]; then
 
 	# Enables/verifies the environment for the build
 	source scl_source enable ${rhpg_pkg}
-	status_rh_postgresql96=$?
+	status_rh_postgresql13=$?
```

Comment:
Please rename `status_rh_postgresql13` to `status_rh_postgresql`, so that it becomes generic.

Reply:
fixed